backend/src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/restaurants_db
+spring.datasource.username=admin
```

Comment:
push default file, without password, but use your own external file for starting project locally